Failed to upgrade @patternfly/react-core due to the way Treeview imported react

**Describe the issue. What is the expected and unexpected behavior?**
I failed to upgrade @patternfly/react-core to 4.50.2 because of the following error with ` import React from 'react'` 
```
node_modules/@patternfly/react-core/dist/js/components/TreeView/TreeViewList.d.ts:1:8 - error TS1259: Module '"/Users/mengting.yan1ibm.com/Workspace7/kui/node_modules/@types/react/index"' can only be default-imported using the 'allowSyntheticDefaultImports' flag

1 import React from 'react';
         ~~~~~

  node_modules/@types/react/index.d.ts:65:1
    65 export = React;
       ~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'allowSyntheticDefaultImports' flag.

node_modules/@patternfly/react-core/dist/js/components/TreeView/TreeViewListItem.d.ts:1:8 - error TS1259: Module '"/Users/mengting.yan1ibm.com/Workspace7/kui/node_modules/@types/react/index"' can only be default-imported using the 'allowSyntheticDefaultImports' flag

1 import React from 'react';
         ~~~~~

  node_modules/@types/react/index.d.ts:65:1
    65 export = React;
       ~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'allowSyntheticDefaultImports' flag.

node_modules/@patternfly/react-core/dist/js/components/TreeView/TreeViewSearch.d.ts:1:8 - error TS1259: Module '"/Users/mengting.yan1ibm.com/Workspace7/kui/node_modules/@types/react/index"' can only be default-imported using the 'allowSyntheticDefaultImports' flag

1 import React from 'react';
         ~~~~~

  node_modules/@types/react/index.d.ts:65:1
    65 export = React;
       ~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'allowSyntheticDefaultImports' flag.
```

All 4.3x components imported react like this:
```typescript
import * as React from 'react'
```
Whereas TreeView (added it seems in the 4.4x series) imports it like this:
```typescript
import React from 'react'
```
The latter forces consuming Typescript projects to have esModuleInterop as a compiler option. Was this an intentional difference? I see that you did not update the other components to use the latter import style. I believe it would be a simple set of changes on your side, to allow for increased flexibility on our (users of patternfly) side. 

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
<!-- PatternFly-React Codesandbox template: https://codesandbox.io/s/serverless-cherry-q9t3f -->
npm install @patternfly/react-core

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
This is a bug blocking me

**What is your product and what release version are you targeting?**
my product: https://github.com/IBM/kui
target version: v4.50.2

--- a/packages/react-charts/src/components/ChartCursorTooltip/ChartCursorFlyout.tsx
+++ b/packages/react-charts/src/components/ChartCursorTooltip/ChartCursorFlyout.tsx
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
 import PropTypes from 'prop-types';
 import { Helpers, CommonProps, Path } from 'victory-core';
 import isPlainObject from 'lodash/isPlainObject';

--- a/packages/react-core/src/components/TreeView/TreeView.tsx
+++ b/packages/react-core/src/components/TreeView/TreeView.tsx
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
 import { TreeViewList } from './TreeViewList';
 import { TreeViewListItem } from './TreeViewListItem';
 

--- a/packages/react-core/src/components/TreeView/TreeViewList.tsx
+++ b/packages/react-core/src/components/TreeView/TreeViewList.tsx
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/TreeView/tree-view';
 import { TreeViewSearch } from './TreeViewSearch';

--- a/packages/react-core/src/components/TreeView/TreeViewSearch.tsx
+++ b/packages/react-core/src/components/TreeView/TreeViewSearch.tsx
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/TreeView/tree-view';
 import formStyles from '@patternfly/react-styles/css/components/FormControl/form-control';

--- a/packages/react-inline-edit-extension/src/components/ConfirmButtons/ConfirmButtons.tsx
+++ b/packages/react-inline-edit-extension/src/components/ConfirmButtons/ConfirmButtons.tsx
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import { CancelButton } from '../CancelButton';
 import { ConfirmButton } from '../ConfirmButton';

